packages/notifi-react-card/lib/components/intercom/FetchedStateCard.tsx

```diff
@@ -0,0 +1,146 @@
+        <div
+          className={clsx(
+            'NotifiInputSeparator__container',
+            'NotifiIntercomInputSeparator__container',
+            inputSeparators?.emailSeparator?.classNames?.container,
+          )}
+        >
+          <div
+            className={clsx(
+              'NotifiInputSeparator__content',
+              inputSeparators.emailSeparator.classNames?.content,
+            )}
+          >
+            {inputSeparators?.emailSeparator?.content}
+          </div>
+        </div>
+      ) : null}
+      <NotifiSmsInput
+        disabled={false}
+        classNames={classNames?.NotifiSmsInput}
+        allowedCountryCodes={allowedCountryCodes}
+        copy={{ label: inputLabels?.sms }}
+        intercomSmsInputStyle={'NotifiIntercomSmsInput__container'}
+        intercomSmsDropdownContainerStyle={
+          'NotifiIntercomSmsInput__dropdownContainer'
+        }
+        intercomSmsDropdownSelectStyle={
+          'NotifiIntercomSmsInput__dropdownSelect'
+        }
+      />
+      {inputSeparators?.smsSeparator?.content ? (
+        <div
+          className={clsx(
+            'NotifiInputSeparator__container',
+            'NotifiIntercomInputSeparator__container',
+            inputSeparators?.smsSeparator?.classNames?.container,
+          )}
+        >
+          <div
+            className={clsx(
+              'NotifiInputSeparator__content',
+              inputSeparators.smsSeparator.classNames?.content,
+            )}
+          >
+            {inputSeparators?.smsSeparator?.content}
+          </div>
+        </div>
+      ) : null}
+      <NotifiTelegramInput
+        disabled={false}
+        classNames={classNames?.NotifiTelegramInput}
+        copy={{ label: inputLabels?.telegram }}
+        intercomTelegramInputStyle={'NotifiIntercomTelegramInput__container'}
+      />
+      {inputSeparators?.telegramSeparator?.content ? (
+        <div
+          className={clsx(
+            'NotifiInputSeparator__container',
+            'NotifiIntercomInputSeparator__container',
+            inputSeparators?.smsSeparator?.classNames?.container,
+          )}
+        >
+          <div
+            className={clsx(
+              'NotifiInputSeparator__content',
+              inputSeparators.telegramSeparator.classNames?.content,
+            )}
+          >
+            {inputSeparators?.telegramSeparator?.content}
+          </div>
+        </div>
+      ) : null}
+      <div
+        className={clsx('BrowserAlertToggle__container', classNames?.container)}
+      >
+        <div className={clsx('BrowserAlertToggle__label', classNames?.label)}>
+          Opt in to browser alerts
+        </div>
+
+        <NotifiToggle
+          classNames={classNames?.toggle}
+          disabled={false}
+          checked={checked}
+          setChecked={setChecked}
+          intercomToggleStyle={'NotifiIntercomToggle__input'}
+        />
+      </div>
+    </div>
+  );
+
+  return <>{contents}</>;
```

Comment:
oo is the jsx fragment necessary? this looks like a copy and paste of the other fetchedStateCard. is that right?

Reply:
updated